.github/workflows/versioning.yml

```
@@ -2,10 +1,13 @@
+# Workflow for managing versioning, tagging, and temporary branches
 ---
-name: Bump Version and Tag
+name: Version Bump and Tag
 
 on:  # yamllint disable-line rule:truthy
   push:
     branches:
       - main
+    commit-message:
```

actionlint (1.7.4)

unexpected key "commit-message" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)